upstairs/src/info.rs

```diff
@@ -28,7 +28,8 @@ pub async fn start_info(
     let config_dropshot = ConfigDropshot {
         bind_address: addr,
         request_body_max_bytes: 1024,
-        tls: None,
+        // tls: None,
+        ..Default::default()
```

Comment:
I changed this because my propolis checkout wouldn't compile without it. TODO for me - pull latest propolis code and try to compile again.